unit1/lesson12.rst

```diff
@@ -0,0 +1,41 @@
+Unit 1 - Lesson 12 Learning Objectives
+=====================================
+
+Goals
+-----
+
+- Create static HTML pages using a variety of different tags inside the body.
+- Create an account on Github and create hello world repository.
+- Understand what Git is and why programmers use it.
+
```

Comment:
Would it be appropriate to address semantic html here, or in a future lesson?

Reply:
I'm 100% on board with teaching semantic HTML, but it feels like a topic that requires more nuance than we have time for in 1-2 classes that also introduce a lot of other topics. My instinct is that we can briefly mention the topic and provide a link to something like [HTML 5 Doctor](http://html5doctor.com/element-index/). If we build a front-end focused unit 2, that would be the place to cover semantic HTML properly, and in-full.

Reply:
What if we simply begin the HTML training with semantic concepts?  Rather than train with the standard tags, could we begin with <section>, etc. instead of <div>?

Reply:
I think semantic HTML is the way to go. As a beginner, I remember finding it easier to visualize what the end result of my code with semantic HTML. Perhaps the ins and outs of semantic HTML would actually be great booster rocket topic!